.github/workflows/databases.yml

```diff
@@ -132,49 +132,49 @@ jobs:
           DB_CONNECTION: mysql
           DB_USERNAME: root
 
-  # pgsql:
-  #   runs-on: ubuntu-20.04
-
-  #   services:
-  #     postgresql:
-  #       image: postgres:14
-  #       env:
-  #         POSTGRES_DB: forge
-  #         POSTGRES_USER: forge
-  #         POSTGRES_PASSWORD: password
-  #       ports:
-  #         - 5432:5432
-  #       options: --health-cmd=pg_isready --health-interval=10s --health-timeout=5s --health-retries=3
-
-  #   strategy:
-  #     fail-fast: true
-
-  #   name: PostgreSQL 14
-
-  #   steps:
-  #     - name: Checkout code
-  #       uses: actions/checkout@v2
-
-  #     - name: Setup PHP
-  #       uses: shivammathur/setup-php@v2
-  #       with:
-  #         php-version: 8.1
-  #         extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, pdo_pgsql
-  #         tools: composer:v2
-  #         coverage: none
-
-  #     - name: Install dependencies
-  #       uses: nick-invision/retry@v1
-  #       with:
-  #         timeout_minutes: 5
-  #         max_attempts: 5
-  #         command: composer update --prefer-stable --prefer-dist --no-interaction --no-progress
-
-  #     - name: Execute tests
-  #       run: vendor/bin/phpunit tests/Integration/Database --verbose --exclude-group MySQL
-  #       env:
-  #         DB_CONNECTION: pgsql
-  #         DB_PASSWORD: password
+  pgsql:
+    runs-on: ubuntu-20.04
+
+    services:
+      postgresql:
+        image: postgres:14
```

Comment:
Would be cool to test on earlier versions too. Almost nobody is using version 14 yet. ;)

Reply:
I don't think it's necessary to test too many versions. We already have quite a bit of builds. It's not like these major postgres versions are so vastly different that they warrant separate builds.

Reply:
Maybe just test the lowest supported, instead of the highest?

Reply:
Yeah that maybe makes sense.